release_notes.md

```diff
@@ -0,0 +1,16 @@
+# What's New
+
+- Following new pages have been added:
```

Comment:
These notes are same as config wizard. It can't be same here in SP. You can create heading as configuration wizard enhancements and then describe details wrt user experience and not as a technicality of the wizard, i.e. what is removed or added.

Reply:
Pluggable hunt framework details are still missing from RN and usage section